yeat/workflow/aux.py

```diff
@@ -81,66 +81,8 @@ def print_downsample_values(genome_size, avg_read_length, coverage, down, seed):
     print(f"[yeat] random seed for sampling: {seed}")
 
 
-def get_expected_files(config):
```

Comment:
From our paired programming, moved this calculation to the individual classes themself. Much cleaner now!